firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -486,4 +495,19 @@ static long parseTokenExpirationTimestamp(String expiresIn) {
         ? 0L
         : Long.parseLong(expiresIn.substring(0, expiresIn.length() - 1));
   }
+
+  // Read the error message from the response.
+  private String readErrorResponse(HttpURLConnection conn) throws IOException {
```

Comment:
If we are only using this for logging, should we swallow that IOException? Wdyt?

Reply:
Makes sense. Done